### PR TITLE
LPS-12345 | master

### PR DESCRIPTION
**Pull request by yanan**
https://***12345
cc @Tim-Cao

### CODE REVIEW

**Comment by yanan** [comment]
ci:test

**Comment by Tim-Cao** [comment]
ci:stop

**Comment by Tim-Cao** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/training/Training.testcase [hunk 1]
```diff
@@ -0,0 +1,45 @@
+@component-name = "user-login"
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/training/Training.testcase:1]: portal-smoke

File: portal-web/test/functional/com/liferay/portalweb/tests/training/Training.testcase [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+		if ("${testPortalInstance}" == "true") {
+			PortalInstances.tearDownCP();
+		}
+		else {
+			PagesAdmin.tearDownCP();
+
+			Site.tearDownCP();
+
+			WebContent.tearDownCP();
+		}
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/training/Training.testcase:22]: No need

File: portal-web/test/functional/com/liferay/portalweb/tests/training/Training.testcase [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+	}
+
+	@description = "Ensure an administrator can login into portal successfully"
+	@priority = "5"
+	test loginTest {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/training/Training.testcase:27]: LoginTest

**Comment by Tim-Cao** [comment]
ci:close

**Pull request closed without merging**